.drone.yml

```diff
@@ -9,28 +9,21 @@ trigger:
     - main
 
 steps:
-  - &deploy
-    name: cs-dev
-    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+  - name: cs-dev
+    image: quay.io/ukhomeofficedigital/helm:v3.5.4
```

Comment:
This image has a fair amount of vulnerabilities - are they planning on upgrading it?

Reply:
I think we can make some PRs against it, but importantly they should track the kubectl version for us as they update the clusters, there is a drift policy so its not so critical (and it's only drone)

Reply:
https://kubernetes.io/releases/version-skew-policy/